app/scripts/views/sign_up.js

```diff
@@ -197,7 +198,24 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
     },
 
     suggestEmail: function () {
-      mailcheck(this.$el.find('.email'), this.metrics, this.translator, this.window.location.search);
+      var MAILCHECK_ON = 'true';
+      var abChoose;
+      var abData = {
+        isMetricsEnabled: !! this.metrics._able,
+        uuid: this.metrics._uuid
+      };
+
+      var mailcheckParamChoice = Url.searchParam('mailcheck', this.window.location.search);
+      // the window parameter will override any ab testing features
+      if (! _.isUndefined(mailcheckParamChoice)) {
+        abChoose = mailcheckParamChoice === MAILCHECK_ON;
+      } else {
+        abChoose = this._able.choose('mailcheckEnabled', abData);
+      }
+
+      if (abChoose) {
```

Comment:
Instead of `abChoose`, a name like `isMailcheckEnabled` conveys the meaning better.